ElementX/Sources/Other/Logging/TracingConfiguration.swift

```diff
@@ -27,32 +26,32 @@ struct TracingConfiguration {
                 return "Debug"
             case .trace:
                 return "Trace"
-            case .custom:
-                return "Custom"
             }
         }
         
-        fileprivate var rawValue: String {
-            switch self {
-            case .error:
-                return "error"
-            case .warn:
-                return "warn"
-            case .info:
-                return "info"
-            case .debug:
-                return "debug"
-            case .trace:
-                return "trace"
-            case .custom(let filter):
-                return filter
+        static func < (lhs: TracingConfiguration.LogLevel, rhs: TracingConfiguration.LogLevel) -> Bool {
+            switch (lhs, rhs) {
+            case (.error, _):
+                true
+            case (.warn, .error):
+                false
+            case (.warn, _):
+                true
+            case (.info, .error), (.info, .warn):
+                false
+            case (.info, _):
+                true
+            case (.debug, .info), (.debug, .warn), (.debug, .error):
```

Comment:
Nit: To keep it consistent with the one above.
```suggestion
            case (.debug, .error), (.debug, .warn), (.debug, .info):
```

Reply:
Right, good point! 😅